portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsUserCollaboration.testcase

```diff
@@ -123,6 +123,111 @@ definition {
 		PublicationsPermissions.assertEditorPermissions(publicationName = "Publication Name 2");
 	}
 
+	@description = "This is a use case for LPS-195716. Can enable and disable share access link."
+	@priority = 3
+	test CanEnableAndDisableShareAccessLink {
+		property custom.properties = "feature.flag.LPS-187436=true";
+
+		task ("Create Web Content and Publication") {
+			JSONPublications.addPublication(publicationName = "Publication Name");
+
+			JSONPublications.selectPublication(publicationName = "Publication Name");
+
+			JSONWebcontent.addWebContent(
+				content = "WC WebContent Content",
+				groupName = "Guest",
+				title = "WC WebContent Title");
+		}
+
+		task ("Go to Publication Review Changes") {
+			PublicationsNavigator.gotoReviewChanges(
+				gotoPublicationsAdmin = "true",
+				publicationName = "Publication Name");
+
+			PublicationsChanges.viewChanges(
+				changeCategory = "Web Content Article",
+				changeSite = "Guest",
+				changeTitle = "WC WebContent Title",
+				changeType = "Added",
+				changeUser = "Test Test");
+		}
+
+		task ("Enable Share Publication Link and copy URL") {
+			Publications.enablePublicationSharing(key_toggleSwitchLabel = "Allow anyone with the link to view this publication.");
```

Comment:
After you updated the locator the key_toggleSwitchLabe is not necessary, can you remove it?